lux/extensions/slack_notifier/slacknotifier.py

```diff
@@ -0,0 +1,71 @@
+import lux
+import json
+import logging
+import pycurl
+from collections import OrderedDict
+
+
+class SlackHandler(logging.StreamHandler):
+    
+    '''Custom handler inheriting from StreamHandler to retain the
+    formatter functionality.
+    Overriding the emit function to use pycurl to send messages to slack
+    (as it turned out to be the most robust option).
+    '''
+    c = pycurl.Curl()
+    c.setopt(c.URL, "https://hooks.slack.com/services/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")
+
+    def emit(self, record):
+        self.c.setopt(self.c.POSTFIELDS, self.format(record))
+        self.c.perform()
+
+
+class Extension(lux.Extension):
+    
+    '''Adding a slackhandler to all loggers so that the messages that we
+    get forwarded to Slack.
+    '''
+    slackhandler = SlackHandler()
+    slackhandler.level = 30
+    slackhandler.setFormatter(logging.Formatter(json.dumps(OrderedDict([
+                        ("text", '%(name)s - %(message)s'),
+                        ("channel", "#bmll-api-logs"),
+                        ("username", "bmll-api notifier"),
+                        ("icon_emoji", ":heavy_exclamation_mark:")]))))
+    
+    
+    #filehandler currently left commented out for testing purposes                        
+    #file gets saved in bmll-api folder    
+    #filehandler = logging.FileHandler("testfile.log")
+    #filehandler.level = 10
+    #filehandler.setFormatter(logging.Formatter(json.dumps(OrderedDict([
+    #                    ("text", '%(name)s - %(message)s'),
+    #                    ("channel", "#bmll-api-logs"),
+    #                    ("username", "bmll-api notifier"),
+    #                    ("icon_emoji", ":heavy_exclamation_mark:")]))))
+
+
+
+    def on_start(self, app, server):
+        #server.logger.parent.addHandler(self.filehandler)
```

Comment:
Like stated above, all references to filehandler will be removed after introducing changes suggested in the review (taking URL, level, formatter from config etc.).